ipa-core/src/ff/prime_field.rs

```diff
@@ -428,6 +434,65 @@ mod fp32bit {
     }
 }
 
+mod fp61bit {
+    field_impl! { Fp61BitPrime, u64, u128, 61, 2_305_843_009_213_693_951
+    }
```

Comment:
Did rustfmt insist on this, or would it tolerate putting it back on the previous line?